app/javascript/elm/src/Main.elm

```diff
@@ -240,7 +240,13 @@ update msg model =
             ( { model | isHttping = isHttpingNow }, Cmd.none )
 
         ToggleIndoor ->
-            ( { model | isIndoor = not model.isIndoor }, Ports.toggleIndoor (not model.isIndoor) )
+            if model.isIndoor then
+                ( { model | isIndoor = not model.isIndoor }, Ports.toggleIndoor (not model.isIndoor) )
```

Comment:
Since `isIndoor` must be True I'd put `False` when updating the model. It's less cryptic than `not`. The same for the else branch

Reply:
yeah was considering it. I'll do it directly on experimental